src/main/resources/static/css/admin.css

```
@@ -300,6 +300,132 @@ body {
+  font-weight: 700;
+  color: var(--text);
+}
+.admin-modal .modal-body { padding: 20px 24px; }
+.admin-modal .modal-footer {
+  padding: 16px 24px 20px;
+  border-top: 1px solid var(--border);
+  gap: 8px;
+}
+.btn-close-sm { width: 24px; height: 24px; }
+
+.form-group-admin {}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)